content/articles/haven-1-2-0.Rmd

```diff
@@ -0,0 +1,36 @@
+---
+title: "haven 1.2.0"
```

Comment:
Decided to make it 1.1.2 since nothing major happened.

Reply:
Updated